Jenkinsfile

```diff
@@ -171,8 +171,9 @@ stage('unit-tests') {
     linux: doDockerBuild('linux', true, false),
     linux_sync: doDockerBuild('linux', true, true),
     android: doAndroidDockerBuild(),
-    macos: doBuild('osx', 'macOS', false),
-    macos_sync: doBuild('osx', 'macOS', true),
+    macos: doBuild('osx', 'macOS', false, ''),
+    macos_1_x: doBuild('osx', 'macOS', false, '-1.x'),
```

Comment:
I started out trying to do that but it turned out to be more involved since the sync-enabled tests build core/sync from source and v2.8.6 doesn't have the cmake build system.